### PR TITLE
Plotsignal

### PR DESCRIPTION
**Pull request by stephenwhitmarsh**

### CODE REVIEW

File: module/plotsignal/plotsignal.py [hunk 1]
```diff
@@ -1,16 +1,40 @@
+# Plotsignal plots raw and spectral data from the buffer and allows
+# interactive selection of frequency bands for further processing
+#
+# Plotsignal is part of the EEGsynth project (https://github.com/eegsynth/eegsynth)
+#
+# Copyright (C) 2007 EEGsynth project
```
Comment (robertoostenveld) [inline module/plotsignal/plotsignal.py]: je leeft in een vorig decennium ;-)

**Comment by stephenwhitmarsh** [comment]
arg... :-)


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by stephenwhitmarsh** [comment]
nieuwe pull request gedaan

On 14 November 2017 at 09:07, Stephen Whitmarsh <stephen.whitmarsh@gmail.com

File: module/plotsignal/plotsignal.py [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
 #!/usr/bin/env python
```
Comment (robertoostenveld) [inline module/plotsignal/plotsignal.py]: this should be all the way at the start of the file. Please change and commit (to this branch)

File: module/plotsignal/plotsignal.py [hunk 3]
```diff
@@ -121,8 +145,8 @@ def butter_lowpass_filter(data, lowcut, fs, order=9):
 redright = [];
 blueleft = [];
 blueright = [];
-fft = [];
-fft_old = [];
+FFT = [];
```
Comment (robertoostenveld) [inline module/plotsignal/plotsignal.py]: I think I have also seen this change in your other PR regarding the GNU licenses...

**Comment by stephenwhitmarsh** [comment]
AARRRGGG... okay... one learns the painful way.

**Pull request closed without merging**